prog/postgres/postgres_timeline_nexus.rb

```diff
@@ -63,7 +63,38 @@ def before_run
 
   label def destroy
     decr_destroy
+    destroy_blob_storage if postgres_timeline.blob_storage_endpoint
```

Comment:
Let's not do this for now. Because we didn't implement purging expired backups and if there is an object in the blob storage, this call fails.